internal/controller/nodeagent.go

```diff
@@ -628,56 +628,25 @@ func (r *DataProtectionApplicationReconciler) customizeNodeAgentDaemonset(ds *ap
 }
 
 func (r *DataProtectionApplicationReconciler) ReconcileFsRestoreHelperConfig(log logr.Logger) (bool, error) {
```

Comment:
would add a comment that this is for 1.4 and lower (and we can remove in 1.6, right?)

Reply:
I'll add the comment. As for removing in 1.6, that depends on whether there will ever be a case where someone upgrades from 1.4 to 1.6 without running the operator at 1.5 in-between.